features/leeftijd_bepaling.feature

```diff
@@ -1,116 +1,100 @@
 # language: nl
 
-@proxy @post-assert
 Functionaliteit: LeeftijdBepaling
   Bepaal de leeftijd van de persoon
 
   Rule: bij een volledig bekende geboortedatum wordt de leeftijd in jaren geleverd
 
-    @skip-verify    
+    @skip-verify
```

Comment:
Scenario's met @skip-verify worden niet uitgevoerd tijdens npm run specs:verify. Dat is helder, toch? Omdat er in deze feature zoveel scenario's zijn die niet kunnen worden uitgevoerd, heb ik onder dev ook leeftijd scenario's toegevoegd